CONTRIBUTING.md

```diff
@@ -115,22 +115,6 @@ checks as the rest of the project. `bin/rake lint` will give you feedback in
 this regard. You can check & fix style issues by running each linter
 individually. Run `bin/rake -T lint` to see the available linters.
 
-### Add a changelog entry
-
-If your PR includes user-observable changes, you'll be asked to add a changelog
-entry following the existing changelog format.
-
-The changelog format is the following:
-
-* One line per PR describing your fix or enhancement.
-* Entries end with a dot, followed by "[#pr-number] by [@github-username]".
-* Entries are added under the "Unreleased" section at the top of the file, under
-  the "Bug Fixes" or "Enhancements" subsection.
-* References to github usernames and pull requests use [shortcut reference
-  links].
-* Your github username reference definition is included in the correct
-  alphabetical position at the bottom of the file.
-
```

Comment:
In the **Shipping a release** section of this document, does anything there need to be updated to account for the overall changes in this PR? Since I haven't shipped a release before I wasn't sure if something changed or additional we'd need to document there for myself or other maintainers.

Reply:
The new changelog generation logic was added to the `bin/rake release:prepare_<minor|major|patch>` tasks, so those tasks now also automatically generate the changelog for the release. I tweaked bullet points in that section to catch up with that :+1:.